### DISCUSSION TITLE
test discussion

### DISCUSSION THREAD

**Discussion by kidayasuo** [w3c/jlreq-d#47] [General]
# test conversation
### test

test edit (@himorin )

**Comment by kidayasuo** [comment]
This is a comment.

**Reply by kidayasuo** [reply]
reply to the first comment.

**Reply by kidayasuo** [reply]
reply to the reply

**Reply by r12a** [reply]
we may need to check whether changes to the discussion threads are sent out with the notification emails.  @xfq could you look into that?

**Reply by kidayasuo** [reply]
I received a notification for your comment, but it might be because it's a reply to my comment.

**Reply by xfq** [reply]
> we may need to check whether changes to the discussion threads are sent out with the notification emails. @xfq could you look into that?

It doesn't look like it's supported yet.

See https://github.com/dontcallmedom/github-notify-ml/issues/67

**Reply by r12a** [reply]
@xfq Would you be able to contact Dom and find out whether he can enable notifications for discussions, including both comments and replies ?  It will be important for that to work if we use discussion threads.  You could also indicate to him that it's fairly urgent.

**Reply by xfq** [reply]
Will do.

**Comment by kidayasuo** [comment]
and the second comment.

**Comment by KobayashiToshi** [comment]
ここに書けばいいんですね．

**Reply by himorin** [reply]
icon with upward arrow (?) seems not functioning???

**Reply by xfq** [reply]
> icon with upward arrow (?) seems not functioning???

Seems to work for me.

**Reply by r12a** [reply]
/me wonders what that up arrow icon means...

**Reply by xfq** [reply]
It's a "vote" button. See:

* https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/participating-in-a-discussion#upvoting-a-comment
* https://github.com/orgs/community/discussions/6174

**Comment by xfq** [comment]
Some examples of other places in W3C where GitHub Discussions is used:

* [wai-mission](https://github.com/w3c/wai-mission/discussions)
* [dapt](https://github.com/w3c/dapt/discussions)
* [low-vision-a11y-tf](https://github.com/w3c/low-vision-a11y-tf/discussions)
* [wcag2ict](https://github.com/w3c/wcag2ict/discussions)
* [secure-payment-confirmation](https://github.com/w3c/secure-payment-confirmation/discussions/259)
* [web-audio-api](https://github.com/WebAudio/web-audio-api/discussions)
* [design-reviews](https://github.com/w3ctag/design-reviews/discussions)
* [gpuweb](https://github.com/gpuweb/gpuweb/discussions)
* [open-ui](https://github.com/openui/open-ui/discussions)

Unicode also uses GitHub Discussions in [icu4x](https://github.com/unicode-org/icu4x/discussions).

**Comment by himorin** [comment]
On 2023/11/30 12:47, himorin wrote:
> icon with upward arrow (?) seems not functioning???

this is a test for replying via email, on e.g. where comment will be appeared.
メールで返したらどこにつくかのテスト

**Comment by r12a** [comment]
@kidayasuo , it appears that we won't be able to receive notifications about new discussions, or comments/replies on those discussions, in the same way as for issues (ie. in the daily digests). As an idea for how to make the best of the situation, i just created a new 'discussion' at https://github.com/w3c/jlreq-d/discussions/50 which i have pinned to the top of https://github.com/w3c/jlreq-d/discussions.  It makes recommendations for people starting a discussion to announce that and encourage others to subscribe to it.  Do you think this looks useful?

**Reply by kidayasuo** [reply]
Yes, thank you. I think your suggestion is very useful. We'll discuss it at today's call.

**Comment by himorin** [comment]
closing as finished testing.
